lib/license_finder/platform.rb

```diff
@@ -1,11 +1,11 @@
 module LicenseFinder
   module Platform
     def self.darwin?
-      RUBY_PLATFORM =~ /darwin/
+      !(RUBY_PLATFORM =~ /darwin/).nil?
     end
 
     def self.windows?
-      RUBY_PLATFORM =~ /mswin|cygwin|mingw/
+      !(RUBY_PLATFORM =~ /mswin|cygwin|mingw/).nil?
```

Comment:
These changes seem acceptable to me ... but I'm not sure I see the **advantage**.

Can you help me understand what the compelling reason is that drove you to submit this pull request?